pulp_smash/tests/rpm/cli/test_upload_and_remove_langpacks.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+"""Tests that perform langpacks upload and removal."""
+from __future__ import unicode_literals
+
+import unittest2
+
+from pulp_smash import cli, config, utils
+from pulp_smash.tests.rpm.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class UploadAndRemoveLangpacksTestCase(unittest2.TestCase):
+    """Upload langpacks and remove it through pulp-rpm.
+
+    This test case targets `Pulp Smash #270`_. How do we test this?
```

Comment:
What about: `This test case targets `Pulp Smash #270`_. The test steps would be as follows:` ?